### DISCUSSION TITLE
What does "StopAt" means in techcnical terms in the context of image prompt influence and pyracanny etc?

### DISCUSSION THREAD

**Discussion by stephantual** [lllyasviel/Fooocus#1603] [Q&A]
I know what weight does, and found a sweetspot for my photo (it's on a case by case basis), but stop at escapes me. At 0 it's basically just the AI doing whatever it wants, at max it's hyper detailed (HDR looking) monstrosity. I would love to undertand what it's supposed to be an abstraction of in technical terms :) Thank you!

**Comment by kpagcha** [comment]
I am merely an user so I don't know the technical aspect, but it seems the "stop at" parameter sets at which iteration percentage to stop taking the image as prompt. So for 30 iterations and "stop at" 0.5, it will take the image prompt for the first 15 iterations (at the set weight, of course), then move on with just the prompt (and the image it has so far, I guess). That's purely my take my from experience, I don't actually know if it works this way.

**Reply by stephantual** [reply]
ah yes you're absolutely correct. It seems that unless it's turned all the way up, it doesn't do very much though. Some documentation on recommended values, expecially if we're facing a situation where everything is bundled around the middle of a bell curve on the slider, would be welcomed :)
Thank you!
